<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color: #ffffff;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>500</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: #4a34cd;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPlainTextEdit" name="text">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>461</width>
        <height>208</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPlainTextEdit{
	background-color: #ffffff;
	border-radius: 10px;
	color: black;
	padding: 10px;
}</string>
      </property>
      <property name="plainText">
       <string/>
      </property>
      <property name="placeholderText">
       <string>place your text here...</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>500</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: #ffffff;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="voices_combo_box">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>210</width>
        <height>30</height>
       </rect>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
      </property>
     </widget>
     <widget class="QSlider" name="speed_slider">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>161</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>300</number>
      </property>
      <property name="value">
       <number>150</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="play_button">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: transparent;
	border: 2px solid #4a34cd;
	border-radius: 10px;
	color: #4a34cd;
}

QPushButton:hover{
	background-color: #4a34cd;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
     <widget class="QLabel" name="speed_number">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel{
	color: #4a34cd;
}</string>
      </property>
      <property name="text">
       <string>150</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
